### PR TITLE
Core/Command: Added disable commands.

### PR DESCRIPTION
**Pull request by HeartWell**
**Changes proposed:**

-  Added disable commands.

**Target branch(es):** 1.x

**Tests performed:** Build and tested

**NOTE** You no longer need to squash your commits, on merge we will squash it for you. (GitHub added a new feature)

### CODE REVIEW

File: src/scripts/ScriptLoader.cpp [hunk 1]
```diff
@@ -600,6 +601,7 @@ void AddCommandScripts()
     AddSC_cast_commandscript();
     AddSC_character_commandscript();
     AddSC_debug_commandscript();
+	AddSC_disable_commandscript();
```
Comment (ShinDarth) [inline src/scripts/ScriptLoader.cpp]: indentation

Comment (HeartWell) [inline src/scripts/ScriptLoader.cpp]: I'll fix it later when I'm at the PC.

Comment (HeartWell) [inline src/scripts/ScriptLoader.cpp]: And fix the error travis, I forgot that you do not have debug_mmaps.

**Comment by Viste** [comment]
Tested work fine, but need to fix .reload disables(not work now need restart server to apply changes in table even if use command .disable)

**Pull request merged**